lib/Elevate/Database.pm

```diff
@@ -0,0 +1,60 @@
+package Elevate::Database;
+
+=encoding utf-8
+
+=head1 NAME
+
+Elevate::Database
+
+Helper/Utility logic for database related tasks.
+
+=cut
+
+use cPstrict;
+
+use Elevate::OS ();
+
+use Cpanel::Pkgr ();
+
+use constant MYSQL_BIN => '/usr/sbin/mysqld';
+
+sub is_database_provided_by_cloudlinux ( $use_cache = 1 ) {
+
+    if ($use_cache) {
+        my $cloudlinux_database_installed = cpev::read_stage_file( 'cloudlinux_database_installed', 0 );
+        return $cloudlinux_database_installed if $cloudlinux_database_installed;
```

Comment:
This only returns if it's true. Shouldn't you also return the cache if it's false?